repository-data/application/src/main/resources/hcm-content/content/documents/administration/labels/pages/itinerary.yaml

```
@@ -260,59 +260,61 @@
       hippostd:state: draft
       hippostdpubwf:createdBy: admin
       hippostdpubwf:creationDate: 2019-12-16T14:56:24.091Z
-      hippostdpubwf:lastModificationDate: 2023-02-17T10:42:06.208Z
+      hippostdpubwf:lastModificationDate: 2025-06-06T12:48:34.949+01:00
       hippostdpubwf:lastModifiedBy: admin
       resourcebundle:descriptions: ['', '', '', '', '', '', '', '', '', '', '', '',
         '', '', '', '', '', '', '', '', '', '', '', '', 'This field is to define a
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 36 but found 10

(indentation)